doc/zhCN/user/source/conf.py

```diff
@@ -54,7 +54,7 @@
 release = '2.19-SNAPSHOT'
 
 # Used in build and documentation links
-branch = 'master'
+branch = 'main'
```

Comment:
Not 100% related to this change (but probably 90% there)... but think the release above should be '2.20-SNAPSHOT' no?

Reply:
right, fixed!